add_options.rb

```diff
@@ -41,4 +44,33 @@ def add_music_album
     print ENTR_MSG
     gets
   end
+
+  def add_game
+    puts 'Pleae enter the publish date of the game:'
+    publish_date = enter_date
+    puts 'Is it multi_player:'
+    multi_player = gets.chomp
+    puts 'The date of the last player:'
+    last_player_at = enter_date
+    @games << Game.new(publish_date, multi_player, last_player_at)
+    puts 'Game is added succussfully'
```

Comment:
The UI must show attention to detail. Please be careful not to print typos that the user can see when using the program. 🔎